src/features/classification/ClassifiedBadge.scss

```diff
@@ -13,4 +13,5 @@
     @include bdl-ClassificationBadge-name;
 
     color: $bdl-gray !important; // To inhibit overriding
+    text-transform: uppercase;
```

Comment:
I was under the impression we wanted to avoid this due to accessibility issues.

Reply:
Yes, this was moved from here:
https://github.com/box/box-ui-elements/pull/1378/files#diff-ee5711bf1ff3f3fcd869e4de1f5c6419L17

It will now only apply to the dynamic text. When the localized message is displayed this style will not be in effect.

Reply:
Cool, sounds good. Thanks for confirming!